Source/WebCore/css/StyleProperties.cpp

```diff
@@ -376,6 +376,8 @@ String StyleProperties::getPropertyValue(CSSPropertyID propertyID, Document* doc
         return get2Values(scrollPaddingInlineShorthand());
     case CSSPropertyWebkitTextOrientation:
         return getPropertyValue(CSSPropertyTextOrientation);
+    case CSSPropertyWebkitBackgroundSize:
+        return getPropertyValue(CSSPropertyBackgroundSize);
```

Comment:
That's not good, there is no round-trip:

```js
element.style.backgroundSize = "100px"; // behaves as "100px auto"
element.style.webkitBackgroundSize; // "100px", oops!
element.style.webkitBackgroundSize += ""; // changes behavior to "100px 100px"
```

```js
element.style.webkitBackgroundSize = "100px"; // behaves as "100px 100px"
element.style.backgroundSize; // "100px", oops!
element.style.backgroundSize += ""; // changes behavior to "100px auto"
```

And I guess this was already wrong, but please fix gCS while you are at it:

```js
element.style.backgroundSize = "100px";
getComputedStyle(element).webkitBackgroundSize; // "100px", oops!
```

This is OK:
```js
element.style.webkitBackgroundSize = "100px";
getComputedStyle(element).backgroundSize; // "100px 100px"
```

See https://github.com/whatwg/compat/issues/28, either serialize `background-size` and `-webkit-background-size` differently, or always use 2 values. People there seemed to prefer 2 values.

Reply:
Yes, and make the unprefixed property parse like -webkit-background-size if it's web-compatible. https://github.com/whatwg/compat/issues/28#issuecomment-700427017

Reply:
Changing the parsing is out-of-scope for this change, and would carry more web compat risk.

I have changed serialization to always use two values.